src/test/librbd/mock/MockImageState.h

```diff
@@ -30,6 +30,8 @@ struct MockImageState {
 
   MOCK_METHOD2(register_update_watcher, int(UpdateWatchCtx *, uint64_t *));
   MOCK_METHOD2(unregister_update_watcher, void(uint64_t, Context *));  
```

Comment:
@smithfarm There seems a whitespace at the end of this line, removed in the master, which I think caused the conflict.